issue_metrics.py

```diff
@@ -119,6 +122,20 @@ def get_per_issue_metrics(
                 author=issue.user["login"],  # type: ignore
             )
 
+            # Extract assignee information from the issue
+            issue_dict = issue.issue.as_dict()  # type: ignore
+            assignee = None
+            assignees = []
+
+            if issue_dict.get("assignee"):
+                assignee = issue_dict["assignee"]["login"]
+
+            if issue_dict.get("assignees"):
+                assignees = [a["login"] for a in issue_dict["assignees"]]
+
```

Comment:
Confirmed issue get response has assignee and assignees in [API docs](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#get-an-issue)